charts/quickstart/charts/redis/templates/deployment.yaml

```
@@ -0,0 +1,53 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "redis.fullname" . }}
+  labels:
+    {{- include "redis.labels" . | nindent 4 }}
+spec:
+  {{- if not .Values.autoscaling.enabled }}
+  replicas: {{ .Values.replicaCount }}
```

yamllint (1.35.1)

[warning] wrong indentation: expected 0 but found 2

(indentation)